content/en/docs/Tasks/secrets.md

```diff
@@ -20,14 +20,6 @@ Beyond this, the keys used to encrypt the Cloud Controller Database
 secrets of the KubeCF deployment. This means that the general process
 explained above __does not apply__ to them.
 
-
-## Secret Rotation
-
-Note, this document explains the general rotation of secrets.
-
-The instructions to rotate the CCDB keys specifically are in
-[a separate document](secret_rotation.md).
```

Comment:
Did it move here? https://github.com/cloudfoundry-incubator/kubecf-docs/blob/master/content/en/docs/Tasks/secrets.md

Reply:
I've moved it here under one page split into paragraphs :paintbrush:

Reply:
this is a leftover of https://github.com/cloudfoundry-incubator/kubecf-docs/commit/0e72670c276dec255f2ffc0bfa6ef75c1e1b68cd